### ISSUE TITLE
Support for multilingual pages

### ISSUE THREAD

**Issue by chucker** [trenz-gmbh/wikidown#53]
This will require some brainstorming.

## Goal

Translating an entire wiki is probably impractical, but it should be possible for individual pages to be available in multiple languages.

## Design idea

* If a page is named `My-Document.en-us.md` instead of `My-Document.md`, consider that the US English version.
* If this matches the browser's `Accept-Language` header, default to showing that.
* If additional languages exist, show flags in the top right to switch to those, e.g. a German flag for `My-Document.de-de.md`.

## Considerations:

* will this suck to use in Azure DevOps itself?
* how does the search know which language to filter for?

**Comment by chucker** [comment]
Yes, but each language's index should also contain the fallback pages, where necessary.

**Comment by chucker** [comment]
I'm confused how this differs from #8. :)

**Comment by ricardoboss** [comment]
You tell me, you're the author of both :D

**Comment by chucker** [comment]
I know 😂
